processor/samplingprocessor/probabilisticsamplerprocessor/probabilisticsampler_test.go

```diff
@@ -462,16 +460,16 @@ func genRandomTestData(numBatches, numTracesPerBatch int, serviceName string, re
 			rs.Resource().Attributes().InsertString("string", "yes")
 			rs.Resource().Attributes().InsertInt("int64", 10000000)
 			rs.InstrumentationLibrarySpans().Resize(1)
+			ils := rs.InstrumentationLibrarySpans().At(0)
+			ils.Spans().Resize(numTracesPerBatch)
 
 			for j := 1; j <= numTracesPerBatch; j++ {
-				span := pdata.NewSpan()
-				span.InitEmpty()
+				span := ils.Spans().At(j - 1)
```

Comment:
why do not fix the for loop instead? :) `for j := 0 ....`

Reply:
Done.